lib/Controller/FileCSVController.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @copyright Copyright (c) 2017 Arawa
+ *
+ * @author 2021 Baptiste Fotia <baptiste.fotia@arawa.fr>
+ * @author 2021 Cyrille Bollu <cyrille@bollu.be>
```

Comment:
You can replace Cyrille by your name with your email address, please ? :slightly_smiling_face: